### ISSUE TITLE
Suggestion Title

### ISSUE THREAD

**Issue by Tamamo-no-Bae** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#1464]
**Describe your suggestion in full detail below:**

Currently the first holder of Felo'melorn in the TFT startdate is Kael'thas.
It should historically have been Anasterian, and then Kael'thas inherited it upon the fall of Quel'thalas in 603.

**Comment by zumbak04** [comment]
> Currently the first holder of Felo'melorn in the TFT startdate is Kael'thas.
> It should historically have been Anasterian, and then Kael'thas inherited it upon the fall of Quel'thalas in 603.

It's how the artifacts spawn using the same ways Paradox does. So most of the artifacts spawn with a wrong original owner except a few ones like Frostmourne, Doomhammer because we can't use the "Paradox" way there.
So, yeah, I wouldn't change it, the Paradox way is simple and more bug-proof.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79902859-3704ba80-8423-11ea-9cfa-d9733ef79ad7.png)

</details>

**Comment by BurnedCrusader** [comment]
I agree with Zumbak. Let's just leave as it is, cause to make it work someone will need to go though headacke for just a cosmic change that no one really cares about.
